src/main/java/counter/Counter.java

```diff
@@ -0,0 +1,20 @@
+package counter;
+
+import org.apache.log4j.Logger;
+
+public class Counter {
+    public static final int START = 0;
+    public static final int END = 100;
+    private static final Logger logger = Logger.getLogger(Counter.class);
+    private int value;
+
+    public Counter(int value) {
+        this.value = value;
+    }
+
+    public void iterate() {
```

Comment:
create method increment that will add 1 to value and call it from the thread (you will have while loop in a thread)